module/importer/src/Infrastructure/Action/TextareaAttributeImportAction.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Infrastructure\Action;
+
+use Ergonode\Attribute\Domain\Entity\Attribute\TextareaAttribute;
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode;
+use Ergonode\Attribute\Domain\ValueObject\AttributeScope;
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use Ergonode\Importer\Domain\Command\Attribute\ImportTextareaAttributeCommand;
+use Ergonode\Importer\Infrastructure\Exception\ImportException;
+use Ergonode\SharedKernel\Domain\Aggregate\AttributeId;
+
+class TextareaAttributeImportAction extends AbstractAttributeImportAction
+{
+    /**
+     * @throws ImportException
+     */
+    public function action(ImportTextareaAttributeCommand $command): void
+    {
+        $this->validate($command);
+        $attribute = $this->updateExistingAttribute($command);
+        if (!$attribute) {
+            $attribute = new TextareaAttribute(
+                AttributeId::fromKey($command->getCode()),
+                new AttributeCode($command->getCode()),
+                new TranslatableString($command->getLabel()),
+                new TranslatableString($command->getHint()),
+                new TranslatableString($command->getPlaceholder()),
+                new AttributeScope($command->getScope()),
+                (bool) $command->getParameter(TextareaAttribute::RICH_EDIT)
```

Comment:
What if the parameter does not exist?

I suggest adding to validate in this param